### PR TITLE
pkg-config include as builddependencies

### PR DESCRIPTION
**Pull request by jrbosch**
pkg-config is needed in systems like Debian because if I don't include it an error occurs in the configuration step

### CODE REVIEW

**Comment by hajgato** [comment]
@jrbosch check PR #2743 and related PRs.

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@jrbosch Thank you for your contribution!

As @hajgato pointed out, there is another open pull request that partially overlaps with this, see #2743.

I'll look into merging that PR first, since there has been quite a bit on cleanup in these X11-related easyconfigs lately by @hajgato, w.r.t. build dependencies, etc.

I will get back to this PR once #2743 is merged in; I can help you with syncing up your branch to get this merge-ready afterwards. Please don't hesitate to ping for an update in case you don't hear back.

**Comment by boegel** [comment]
@jrbosch Please see https://github.com/jrbosch/easybuild-easyconfigs/pull/1 which resolves the merge conflicts that now occur after merging #2743.

File: easybuild/easyconfigs/l/libXdmcp/libXdmcp-1.1.2-foss-2016a.eb [hunk 1]
```diff
@@ -15,8 +15,10 @@ sources = [SOURCE_TAR_GZ]
 source_urls = ['http://xorg.freedesktop.org/archive/individual/lib/']
 
 builddependencies = [
+    ('pkg-config', '0.29')
```
Comment (boegel) [inline easybuild/easyconfigs/l/libXdmcp/libXdmcp-1.1.2-foss-2016a.eb]: @jrbosch I managed to introduce a syntax error here, leading to this error:

```
TypeError: 'tuple' object is not callable
```

The problem is that there's a comma (`,`) missing at the end of this line, so please change this to:

```python
    ('pkg-config', '0.29'),
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2623.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0c046f9c0a0788d2574e739f6f283038 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d403251561fd30d4ed0f2422154b905f for a full test report.

**Comment by boegel** [comment]
Good to go, thanks @jrbosch, and congratulations on your first contribution to EasyBuild! Keep them coming!

**Pull request merged**